### PR TITLE
Tooltip update to reflect current product feature availability

### PR DESCRIPTION
**Pull request by allenfear**
Closes: # n/a

## Summary

**[Get Support](https://pantheon.io/docs/support)** - Bringing the tooltip up to date with current scope. Custom Upstream availability applies to Gold accounts and Localdev was also released since this tooltip was originally published

## Effect

The following changes are already committed:

-
-

## Remaining Work

The following changes still need to be completed: n/a

- [ ] List any outstanding work here

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6288](https://pr-6288--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `52` | `52` | `0` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6288` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/26271/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/26271/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6288--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by EdwardAngert** [comment]
taking this back for a minute - I think I can make those popovers a tiny bit nicer

**Review by michellecolon-pantheon — CHANGES_REQUESTED** [review]
Hi @EdwardAngert - Do you think "integrations" on Line is 37 should be capitalized?

File: source/content/support.md [hunk 1]
```diff
@@ -14,12 +14,33 @@ Pantheon offers a range of Account options that include the features required fo
...
+- **Technical**: Adds support for:
+   - Autopilot
+   - Drupal and WordPress application-level and site support
+   - Identifying problematic modules and plugins
+   - Multidev
+   - Assistance with basic CDN, Terminus, Localdev, and Custom Upstreams
+- **Performance**:
+   - New Relic
+   - Caching
+   - Cloud integrations
```
Comment (michellecolon-pantheon) [inline source/content/support.md:37]: This one :)

**Review by michellecolon-pantheon — APPROVED** [review]
LGTM

**Pull request merged**